File: New Unity Project/Assets/IndieEffects/JS Classes/Negative.js
#pragma strict
import IndieEffects;
/*

				----------Negative----------
When i was playing around with the indie effects motion blur shader, i got 
this effect by accident. enjoy!
*/
@script RequireComponent (IndieEffects)
@script AddComponentMenu ("Indie Effects/Negative")
var fxRes : IndieEffects;

private var ThermoMat : Material;
var shader : Shader;
var noise : float;

function Start () {
	fxRes = GetComponent(IndieEffects);
	ThermoMat = new Material(shader);
}

function Update () {
	ThermoMat.SetTexture("_MainTex", fxRes.RT);
}

function OnPostRender () {
	FullScreenQuad(ThermoMat);
}

File: Indie Effects Git/Assets/IndieEffects/CSharp Classes/Negative.cs
using UnityEngine;

/*

				----------Negative----------
When i was playing around with the indie effects motion blur shader, i got 
this effect by accident. enjoy!
*/
[RequireComponent(typeof(IndieEffects))]
[AddComponentMenu("Indie Effects/Negative C#")]
public class Negative : MonoBehaviour
{
    public IndieEffects fxRes;

    private Material ThermoMat;
    public Shader shader;
    public float noise;

    public void Start () {
	    fxRes = GetComponent<IndieEffects>();
	    ThermoMat = new Material(shader);
    }

    public void Update () {
	    ThermoMat.SetTexture("_MainTex", fxRes.RT);
    }

    public void OnPostRender () {
	    IndieEffects.FullScreenQuad(ThermoMat);
    }
}